pkg/lib/bundle/validate.go

```diff
@@ -45,8 +45,10 @@ type imageValidator struct {
 // PullBundleImage shells out to a container tool and pulls a given image tag
 // Then it unpacks the image layer filesystem contents and pushes the contents
 // to a specified directory for further validation
-func (i imageValidator) PullBundleImage(imageTag, directory string) error {
+func (i imageValidator) PullBundleImage(containerTool, imageTag, directory string) error {
 	i.logger.Debug("Pulling and unpacking container image")
+	i.imageReader = containertools.NewImageReader(containertools.NewContainerTool(containerTool,
```

Comment:
It's a bit confusing to mutate a field on the receiver here, since it's a value type. At this point, it doesn't make sense to keep the `imageReader` field on `imageValidator` at all, and this function doesn't really need to have a receiver.

Is there any reason not to remove this method entirely and call ImageReader.GetImageData directly?